include/gsl/span

```diff
@@ -519,8 +563,8 @@ public:
         typename details::calculate_subspan_type<ElementType, Extent, Offset, Count>::type
     {
         Expects((size() >= Offset) && (Count == dynamic_extent || (Count <= size() - Offset)));
-
-        return {data() + Offset, Count == dynamic_extent ? size() - Offset : Count};
+        using type = typename details::calculate_subspan_type<ElementType, Extent, Offset, Count>::type;
+        return type(data() + Offset, Count == dynamic_extent ? size() - Offset : Count);
```

Comment:
Style: The previous lines used braces to construct the returned `span`, but this uses parentheses. Ditto for the following `span` returns.

Reply:
Cleaned that up. Thanks.